### DISCUSSION TITLE
project.nvim confusion

### DISCUSSION THREAD

**Discussion by dinhlongviolin1** [LazyVim/LazyVim#5161] [Q&A]
I am trying to use project.nvim in lazyextras. It seems to me that it is based on telescope and now lazyvim is using fzflua. Is there any additional setup that I need to do to make it work or I cant use this extension because telescope is not used anymore.

**Comment by gengdz** [comment]
No additional actions are required, LazyVim did all the work for us.
 
 1. enable `util.project` in LazyExtras
 2. use `<leader>fp`
 
 project  extras is based on `LazyVim.pick` , `Telescope` and `fzf-lua` are both specific implementations of pick.

**Reply by dinhlongviolin1** [reply]
Thanks for the clarification. I do see that something pops up when using leader fp but there is nothing on it. I tried using it between multiple git projects to see if it picks up the project but nothing seems to show up.

**Reply by dpetka2001** [reply]
You have to manually enter projects because by default it's configured to [manual_mode = true](https://www.lazyvim.org/extras/util/project). Read the docs in its repo for how to do that or change `manual_mode = false` in your personal configuration. Keep in mind that it was changed to `true` because it was messing with setting the `cwd` and that affected the search with the pickers, so it was left up to the users if they wanted such behavior.
